modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -83,47 +92,80 @@ dynamicPlugins:
             icon: fooIcon
             text: Foo Plugin Page
       menuItems:
-        my-plugin: # matches `path` in `dynamicRoutes`
-          priority: 10 # controls order of plugins under the parent menu item
-          parent: favorites # nests this plugin under the `favorites` parent menu item
-        favorites: # configuration for the parent menu item
-          icon: favorite # icon from RHDH system icons
-          title: Favorites # title for the parent menu item
-          priority: 100 # controls the order of this top-level menu item
+        my-plugin: # <1>
+          priority: 10 # <2>
+          parent: favorites # <3>
+        favorites: # <4>
+          icon: favorite # <5>
+          title: Favorites # <6>
+          priority: 100 # <7>
 ----
---
+  <1> `my-plugin`: Matches `path` in `dynamicRoutes`
+  <2> `priority`: Controls order of plugins under the parent menu item.
+  <3> `parent`: Nests this plugin under the `favorites` parent menu item.
+  <4> `favourites`: Configuration for the parent menu item.
+  <5> `icon`: Displays the `favorite` icon from the {product-very-short} system icons.
+  <6> `title`: Displays the title name for the parent menu item.
+  <7> `priority`: Order of the `favourites` menu item in the sidebar.
```

Comment:
Asciidoc syntax mistake. The two leading empty spaces denote a literal block, which is not what you intend. See [docs.asciidoctor.org/asciidoc/latest/verbatim/literal-blocks#indent-method](https://docs.asciidoctor.org/asciidoc/latest/verbatim/literal-blocks/#indent-method)

```suggestion
<1> `my-plugin`: Matches `path` in `dynamicRoutes`
<2> `priority`: Controls order of plugins under the parent menu item.
<3> `parent`: Nests this plugin under the `favorites` parent menu item.
<4> `favourites`: Configuration for the parent menu item.
<5> `icon`: Displays the `favorite` icon from the {product-very-short} system icons.
<6> `title`: Displays the title name for the parent menu item.
<7> `priority`: Order of the `favourites` menu item in the sidebar.
```